static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.tsx

```diff
@@ -148,11 +150,23 @@ const SampleWidgetCard = styled(motion.div)<{isTable: boolean}>`
   margin: auto;
 `;
 
+const ContainerWithoutSidebar = styled('div')`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  height: 100vh;
```

Comment:
I think this might be unnecessary, I took it out and it looks like it does fine

Reply:
the `left: 0;` seems necessary. when I take it out it's being janky LOL so i'm keeping that in